### PR TITLE
Operator API changes and docs update

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

File: docs/api/operator.yaml [hunk 1]
```diff
@@ -162,50 +218,106 @@ paths:
...
+                type: integer
+                enum: [1, 2, 3]
       responses:
         200:
-          description: The updated parking
+          description: The parking was updated successfully
           schema:
             $ref: '#/definitions/Parking'
         401:
+          description: Unauthorized
```
Comment (mingfeng) [inline docs/api/operator.yaml]: Why not `$ref: '#/responses/Unauthorized'`?

File: parkings/api/operator/parking.py [hunk 2]
```diff
@@ -1,53 +1,22 @@
...
-from parkings.models import Address, Operator, Parking
+from parkings.models import Operator, Parking
 
-from ..common import AddressSerializer, ParkingFilter
+from ..common import ParkingFilter
 
 
 class OperatorAPIParkingSerializer(serializers.ModelSerializer):
     status = serializers.ReadOnlyField(source='get_state')
-    address = AddressSerializer(allow_null=True, required=False)
```
Comment (mingfeng) [inline parkings/api/operator/parking.py]: So parkings have addresses but we don't expose it to the REST API?

**Comment by mingfeng** [comment]
Looks solid to me :+1:

File: parkings/api/operator/parking.py (hunk 2 above)
Comment (tuomas777) [inline parkings/api/operator/parking.py]: Yeah, well, we will probably remove addresses completely, but that is not sure yet, so I started with removing it from Operator API, which is the one that actually matters ATM. :)

**Pull request merged**